src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -68,15 +97,15 @@ static UNK_TYPE D_80AF4118 = 0;
 #include "z_en_ru2_cutscene_data.inc.c"
 
 static EnRu2ActionFunc sActionFuncs[] = {
-    func_80AF2CB4, func_80AF2CD4, func_80AF2CF4, func_80AF2D2C, func_80AF2D6C, func_80AF2DAC, func_80AF2DEC,
-    func_80AF3144, func_80AF3174, func_80AF31C8, func_80AF3604, func_80AF3624, func_80AF366C, func_80AF36AC,
-    func_80AF3BC8, func_80AF3C04, func_80AF3C64, func_80AF3CB8, func_80AF3D0C, func_80AF3D60,
+    EnRu2_Action00, EnRu2_Action01, EnRu2_Action02, EnRu2_Action03, EnRu2_Action04, EnRu2_Action05, EnRu2_Action06,
+    EnRu2_Action07, EnRu2_Action08, EnRu2_Action09, EnRu2_Action10, EnRu2_Action11, EnRu2_Action12, EnRu2_Action13,
+    EnRu2_Action14, EnRu2_Action15, EnRu2_Action16, EnRu2_Action17, EnRu2_Action18, EnRu2_Action19,
 };
 
 static EnRu2DrawFunc sDrawFuncs[] = {
-    func_80AF3F14,
-    func_80AF3F20,
-    func_80AF321C,
+    EnRu2_DrawNothing,
+    EnRu2_DrawSkelAnimeOpa,
+    EnRu2_DrawSkelAnimeFlex,
```

Comment:
I think they can just be `EnRu2_DrawOpa` and `EnRu2_DrawXlu`

I dont think "skelanime" needs to be in the name, it is a given that their model is the thing being drawn. And the important distinction here is that one is opaque and one is translucent, not that one of them is "flex".